packages/ccd-js-gen/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+- Generate `create<ContractName>ParameterWebWallet` and `create<EntrypointName>ParameterWebWallet` functions for constructing the smart contract parameters in the format used by the Concordium Web-Wallet.
+- Minor breaking change: `create<EntrypointName>Parameter` function for contract entries and `create<ContractName>Parameter` function in module are no longer generated when parameter schema type is `Unit`.
+  This function is mainly used internally, but were still exposed.
```

Comment:
Which function?

Reply:
Right, I have tried to rephrase this a bit